.env

```diff
@@ -37,5 +37,5 @@ SULU_ADMIN_EMAIL=example@localhost
 ###< sulu/sulu ###
 
 ###> symfony/mailer ###
-# MAILER_DSN=null://null
+MAILER_DSN=null://null
```

Comment:
phpstan did failed without it as it failed initial some command services but a mailer is required as swiftmailer is not longer registered